discovery_engine_core/providers/src/lib.rs

```diff
@@ -25,18 +25,21 @@
 #![warn(missing_docs, unreachable_pub)]
 #![allow(clippy::missing_errors_doc, clippy::must_use_candidate)]
 
+use chrono::NaiveDateTime;
+use displaydoc::Display as DisplayDoc;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
 mod client;
 mod expression;
 mod filter;
-mod gnews;
-pub mod gnews_client;
-mod newscatcher;
-pub mod newscatcher_client;
+pub mod gnews;
+pub mod newscatcher;
 mod utils;
 
-pub use client::{Article, Client, Error};
-pub use gnews_client::{HeadlinesQuery as GnewsHeadlinesQuery, NewsQuery as GnewsNewsQuery};
-pub use newscatcher_client::{
+pub use client::Client;
+pub use gnews::{HeadlinesQuery as GnewsHeadlinesQuery, NewsQuery as GnewsNewsQuery};
+pub use newscatcher::{
```

Comment:
🔧 this exports the items from those two modules, but at the same time these modules are public anyways, one of those options should be enough.

Reply:
Not code I wrote, and will be changed anyway in follow up PR so wont fix in this PR.